Jenkinsfile

```diff
@@ -48,129 +49,112 @@ pipeline {
+                                sh 'mvn install -B -pl !$DOCS -DskipStatic=true -DskipTests=true $DISABLE_DOWNLOAD_PROGRESS_OPTS -T 1C'
+                                sh 'mvn clean install -B -T 1C -pl !$ITESTS -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                                sh 'mvn install -B -pl $ITESTS -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS'
                             }
                         }
-                    },
-                    windows: {
-                        node('proxmox-windows') {
-                            bat 'git config --system core.longpaths true'
-                            retry(3) {
-                                checkout scm
-                            }
-                            timeout(time: 1, unit: 'HOURS') {
-                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS}', options: [artifactsPublisher(disabled: true), dependenciesFingerprintPublisher(disabled: true, includeScopeCompile: false, includeScopeProvided: false, includeScopeRuntime: false, includeSnapshotVersions: false)]) {
-                                    bat 'mvn install -B -pl !%DOCS% -DskipStatic=true -DskipTests=true -T 1C'
-                                    bat 'mvn clean install -B -T 1C -pl !%ITESTS% -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/%CHANGE_TARGET%'
-                                    bat 'mvn install -B -pl %ITESTS% -nsu'
-                                }
+                    }
+                }
+                stage ('Windows') {
+                    agent { label 'server-2016-large' }
+                    steps {
+                        retry(3) {
+                            checkout scm
+                        }
+                        timeout(time: 1, unit: 'HOURS') {
+                            withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS}', options: [artifactsPublisher(disabled: true), dependenciesFingerprintPublisher(disabled: true, includeScopeCompile: false, includeScopeProvided: false, includeScopeRuntime: false, includeSnapshotVersions: false)]) {
+                                bat 'mvn install -B -pl !%DOCS% -DskipStatic=true -DskipTests=true $DISABLE_DOWNLOAD_PROGRESS_OPTS -T 1C'
+                                bat 'mvn clean install -B -T 1C -pl !%ITESTS% -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/%CHANGE_TARGET% $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                                bat 'mvn install -B -pl %ITESTS% -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
Does this need to be `%DISABLE_DOWNLOAD_PROGRESS_OPTS%` to account for being windows?

Reply:
yes it does ;)